src/component/registry/css-registry.ts

```diff
@@ -25,7 +26,7 @@ export class CssRegistry {
    * @return the registered CSS class names
    */
   getClassNames(bpmnElementId: string): string[] {
-    return this.classNamesByBPMNId.get(bpmnElementId) || [];
+    return Array.from(this.classNamesByBPMNId.get(bpmnElementId) || '');
```

Comment:
I'm not sure that we want to return an array with an empty string, where there is no registered class, but more a empty array.

Reply:
This returns an empty array, we have tests that ensure it, see https://github.com/process-analytics/bpmn-visualization-js/blob/29db4c9c94454169d94bd9c226e59a283d96d5be/test/unit/component/registry/css-registry.test.ts#L24-L32
As the syntax seems to be confusing, I can change it (replace the empty string with something different)

Reply:
Updated in f234b267